index.js

```diff
@@ -183,7 +183,12 @@ function build (opts = {}) {
       }
     })
 
-    const destination = sonic({ dest: opts.destination || 1, sync: false })
+    const destination = sonic({
```

Comment:
Could the user pass in a `SonicBoom` instance instead? This change leads us down the path of having to maintain 1:1 compatibility with all of the `SonicBoom` constructor options.

Leave the current default, but check for `opts.destination is SonicBoom` before it. That will make it a backward compatible change.